test/acceptance/features/steps/openshift.py

```diff
@@ -373,3 +373,13 @@ def get_deployment_with_intermediate_secret_of_given_pattern(self, intermediate_
             else:
                 print(f"No deployment that matches {deployment_name_pattern} found.\n")
         return None
+
+    def delete_service_binding(self, sb_name):
+        (output, exit_code) = self.cmd.run("oc delete ServiceBinding " + sb_name)
+        assert exit_code == 0, "Deleting service binding failed as the exit code is not 0"
+        return output
+
+    def check_secret_existence(self, secret_name):
+        (output, exit_code) = self.cmd.run("oc get secret " + secret_name)
```

Comment:
we can use a pre-existing function https://github.com/redhat-developer/service-binding-operator/blob/master/test/acceptance/features/steps/openshift.py#L118

Reply:
That function is for a type. But here a particular instance of Secret is what is required.